bin/internal/goldens.version

```diff
@@ -1 +1 @@
-1a999092d10a22bc700214b257cd4890c5800078
+54957e160ccd4db92659fccaaaa81631c95bb9d2
```

Comment:
This commit doesn't exist in the master branch -- https://github.com/flutter/goldens/commits/master